src/OrleansRuntime/Messaging/OutboundMessageQueue.cs

```diff
@@ -122,7 +119,6 @@ public bool SendMessage(Message msg)
                 if (stopped)
                 {
                     logger.Info(ErrorCode.Runtime_Error_100115, "Message was queued for sending after outbound queue was stopped: {0}", msg);
-                    return true;
```

Comment:
@jthelin , I am not actually saying that the missing returns are wrong or not.
Your commit comment seemed to imply that you were not trying to change the structure or semantics of this method, merely remove the bool. So my review is simply a structural code check, without checking the impact of the missing return  statements on the semantics.